azurerm/internal/services/monitor/monitor_activity_log_alert_resource.go

```diff
@@ -151,6 +151,49 @@ func resourceMonitorActivityLogAlert() *schema.Resource {
 							Optional:      true,
 							ConflictsWith: []string{"criteria.0.recommendation_category", "criteria.0.recommendation_impact"},
 						},
+						"servicehealth": {
```

Comment:
Could we change this to `service_health`?
```suggestion
						"service_health": {
```

Reply:
Yes fine for me.